src/Platforms/AbstractPlatform.php

```
@@ -35,14 +37,23 @@ public function id(): string {
-    // Noop. Override to inject context-specific settings.
+    // Known interfaces - optimised for speed.
+    if ($this instanceof DrupalContextualizerInterface && $context instanceof Drupal) {
+      // Only the most-derived contextualizeDrupal() runs, so a subclass that
+      // overrides it must call parent::contextualizeDrupal() to keep the
+      // parent's settings.
+      $this->contextualizeDrupal($context);
+      return;
+    }
+
+    // Runtime.
+    $method = 'contextualize' . (new \ReflectionClass($context))->getShortName();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '51', column '38'). (undefined)

(MissingImport)